core/neat/settings/_settings.scss

```diff
@@ -47,7 +47,7 @@ $neat-grid: () !default;
 ///
 /// @property {number (with unit) | string | null} media [null]
 ///   The `@media` definition that is used by the [`grid-media`](#grid-media)
-///   mixin to detirmine the media properties.
+///   mixin to determine the media properties.
```

Comment:
Looks like this was already fixed in https://github.com/thoughtbot/neat/commit/483d14341064420a6b90fcc8fc8dee2d2c2de57f.